docs/reference/hooks.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Hooks
+parent: Reference
```

Comment:
I would suggest having this as a top-level item in the docs tree

Reply:
Done.